packages/typir/src/index.ts

```diff
@@ -22,5 +22,6 @@ export * from './kinds/function-kind.js';
 export * from './kinds/kind.js';
 export * from './kinds/multiplicity-kind.js';
 export * from './kinds/primitive-kind.js';
+export * from './kinds/top-kind.js';
```

Comment:
What about `bottom`/`never`, the empty type.

Reply:
Yes, they are still missing. Let's do it in another PR, since they are not required in LOX as far as I can see.